statsmodels/tsa/arima/estimators/hannan_rissanen.py

```diff
@@ -148,13 +150,7 @@ def hannan_rissanen(endog, ar_order=0, ma_order=0, demean=True,
             res = mod.fit()
             resid = res.resid
             p.sigma2 = res.scale
-            p.ar_params = _stitch_fixed_and_free_params(
-                fixed_ar_or_ma_lags=params_info.fixed_ar_lags,
-                fixed_ar_or_ma_params=params_info.fixed_ar_params,
-                free_ar_or_ma_lags=params_info.free_ar_lags,
-                free_ar_or_ma_params=res.params,
-                spec_ar_or_ma_lags=spec.ar_lags
-            )
+            p.free_ar_params = res.params
```

Comment:
We can now easily set free params like this. Fixed and free params can later together be accessed with, for example, `.ar_params`, so there is no need for this stitching function.